### DISCUSSION TITLE
Getting the exact times of transmissions

### DISCUSSION THREAD

**Discussion by dbakker** [ggerganov/ggwave#56] [Ideas]
I'm absolutely blown away by this project & how robust it works even with noise!

I was wondering, is it possible to find the exact times of transmissions? Like to get the start+end time with ms accuracy?

If that could be consistent even in the presence of noise, it could be very useful for synchronizing audio tracks instead of using clappers/timecode generators.

**Comment by ggerganov** [comment]
Hi and thanks for the nice words!

There has been a recent discussion about estimating the exact time of the transmission: https://github.com/ggerganov/ggwave/issues/46
I am still not sure about the best timing accuracy that can be achieve though - need to experiment

**Comment by franga2000** [comment]
@dbakker sorry to ping you in an old thread like this, but I just found this and a though I'd share that I've been using a ggwave-powered digital slate on short film sets for over a year now with great success. 

I have a simple [web app](https://slate.dev.frangez.me/adhoc.html) which lets me put in the current scene, shot and take. This info, along with a millisecond timestamp, is then played out loud with ggwave. I use an old smartphone taped to the back of the slate ([yes, really](https://user-images.githubusercontent.com/3891092/164077561-b8aab48c-34d1-4e01-908e-098782ae8af4.JPG)) to trigger it after slating the usual way, but I'm working on a custom slate which will integrate the two. 

A program (that I'm currently rewriting, which is why I found this thread only now) then scans my footage folder and reads the information from the sound into a spreadsheet and most importantly renames matching clips in Premiere Pro. Here's a sample report from a one day shoot: [zagreb.csv](https://github.com/ggerganov/ggwave/files/8515906/zagreb.csv) (we didn't use it for every shot, but excluding those it had a 77% success rate).

As for sync, we're not syncing directly with ggwave, but use it to estimate the slate point and then run a regular waveform matching algorithm around it.

**Reply by dbakker** [reply]
Hi @franga2000 thanks for sharing, that looks pretty cool! Looks like you've made a lot of takes with this. 😯 Perhaps you could modify a bluetooth smartphone camera trigger or use a bluetooth enabled arduino to trigger your tablet :thinking: . And have it autoincrement the take.
